### PR TITLE
Rule updated and added

### PR DESCRIPTION
**Pull request by Tuutaans**
### Summary of the Pull Request

This PR updates the `Potential System DLL Sideloading From Non System Locations` and `Screen Capture Activity Via Psr.EXE`

### Changelog
update: Screen Capture Activity Via Psr.EXE - Add `-start` commandline variation
update: Potential System DLL Sideloading From Non System Locations - Add `iernonce.dll`

### Example Log Event

N/A

### Fixed Issues

N/A

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

**Review by nasbench — APPROVED** [review]

**Comment by nasbench** [comment]
@Tuutaans as I said before, please rebase your branch or create a new branch when you submit new rules. This is becoming tedious :)

**Comment by Tuutaans** [comment]
@nasbench, I created new from the existing branch instead of sigma.😂 😂

**Comment by nasbench** [comment]
Why are you deleting the branch????

**Comment by Tuutaans** [comment]
My bad..

File: rules/windows/process_creation/proc_creation_win_psr_capture_screenshots.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
-title: Psr.exe Capture Screenshots
+title: Screen Capture Activity Via Psr.EXE
 id: 2158f96f-43c2-43cb-952a-ab4580f32382
 status: test
-description: The psr.exe captures desktop screenshots and saves them on the local machine
+description: Detects execution of Windows Problem Steps Recorder (psr.exe) a utility used to record the user screen and clicks.
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_psr_capture_screenshots.yml:4]:
```suggestion
description: Detects execution of Windows Problem Steps Recorder (psr.exe), a utility used to record the user screen and clicks.
```

**Review by phantinuss — APPROVED** [review]

**Pull request merged**